app/helpers/pdf_js_helper.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module PdfJsHelper
```

Comment:
I suspect this has aspects that belong Hyku Prime, though "reader/archival" split requires further thought.  But this requires further consideration.